lib/pronto/brakeman/version.rb

```diff
@@ -1,5 +1,5 @@
 module Pronto
   module BrakemanVersion
-    VERSION = '0.11.0'.freeze
+    VERSION = '0.12.0'.freeze
```

Comment:
```suggestion
    VERSION = '0.11.1'.freeze
```
Don't want to bump up to `0.12.x` without syncing across all the gems.

Reply:
👍🏻